Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1143,7 +1143,6 @@ class BrowserViewController: UIViewController {
     func finishEditingAndSubmit(_ url: URL, visitType: VisitType, forTab tab: Tab) {
         urlBar.currentURL = url
         leaveOverlayMode(didCancel: false)
-
```

Comment:
I guess that change is not needed, is it? no blocker though

Reply:
Tried to remove, was causing a new line with space - so decided to leave it as is 🙏